scalastyle-config.xml

```diff
@@ -104,6 +104,23 @@ You can also disable only one rule, by specifying its rule id, as specified in:
         <customMessage>Use Javadoc style indentation for multiline comments</customMessage>
     </check>
 
+    
+    <!-- ================================================================================ -->
+    <!--       rules for enforcing cross-build between Scala 2.12 and 2.13                -->
+    <!-- ================================================================================ -->
+
+    <check level="error" class="org.scalastyle.file.RegexChecker" enabled="true">
+        <parameters>
+            <parameter name="regex">def this\((.*)\) \{</parameter>
```

Comment:
This regex does not catch all white space characters or even repeated " ":
```scala
def this(foo: Type)
{
   // line break instead of a simple space
}
```